02-express-tutorial/public/index.js

```diff
@@ -0,0 +1,44 @@
+console.log("javascript file is linked");
```

Comment:
You can remove all console.log on this file. Usually, they are only useful when debugging. You most likely do not want to print everything to the console.
First, this can make private data visible to the user of the site.
Second, it makes the code harder to follow, now you have to visually guess which parts are relevant to the logic or are just logs.